task/monorail-server/src/bsp/sidecar_bc.rs

```diff
@@ -374,30 +393,21 @@ impl<'a, R: Vsc7448Rw> Bsp<'a, R> {
 
     pub fn phy_vsc8562_init(&mut self) -> Result<(), VscError> {
         if let Some(phy_rw) = &mut self.vsc8562 {
-            // Do a hard power cycle of the PHY; the fine details of sequencing
-            // are handled by the FPGA
-            phy_rw
-                .set_phy_power_enabled(false)
-                .map_err(|e| VscError::ProxyError(e.into()))?;
-            sleep_for(10);
-            phy_rw
-                .set_phy_power_enabled(true)
+            // Request a reset of the PHY.
```

Comment:
```suggestion
            // Request a reset of the PHY.  If we had previously marked the PHY
            // oscillator as bad, then this power-cycles the entire front IO
            // board; otherwise, it only power-cycles the PHY.
```